source/guide_pleroma.rst

```diff
@@ -0,0 +1,279 @@
+.. highlight:: console
+.. author:: Arian Malek
+
+.. tag:: lang-elixir
+.. tag:: microblogging
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/pleroma.svg
+      :align: center
+
+#########
+Pleroma
+#########
+
+.. tag_list::
+
+Pleroma_ is a free, federated social networking server built on open protocols. It is compatible with `GNU Social`_, Mastodon_, and many other ActivityPub implementations.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :lab:`PostgreSQL <guide_postgresql>`
+  * :manual:`Domains <web-domains>`
+  * :manual:`Web Backend <web-backend>`  
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+You should've running a :lab:`PostgreSQL <guide_postgresql>` in the version ``9.6`` or above configured with ``--with-uuid=e2fs``
+
+::
+
+ [isabell@stardust ~]$ ./configure --prefix=$HOME/opt/postgresql/ --with-python PYTHON=/usr/bin/python2 --without-readline --with-uuid=e2fs
+ [isabell@stardust ~]$
+
+In PostgreSQL you need a database with your username as name. Otherwise there is an error when you create the database for pleroma.
+
+.. warning:: Please replace ``<username>`` with your Uberspace username created in :lab:`PostgreSQL <guide_postgresql>`!
+
+::
+
+ [isabell@stardust ~]$ createdb --encoding=UTF8 --owner=<username> <username>
```

Comment:
This is required how I write on line 42:
_In PostgreSQL you need a database with your username as name. Otherwise there is an error when you create the database for pleroma._

Without this database you cant run the generated script later to create pleroma user & database. Maybe a bug?

Reply:
Yep, had the same issue. Seems to be some weird behaviour of postgres.